modules/develop/pages/manage-topics/cloud-topics.adoc

```diff
@@ -0,0 +1,89 @@
+
+== Prerequisites
+
+- xref:get-started:rpk-install.adoc[] v25.3 or later.
+ifndef::env-cloud[]
+- xref:manage:tiered-storage.adoc#set-up-tiered-storage[Set up] and xref:manage:tiered-storage.adoc#enable-tiered-storage[enable Tiered Storage] on your Redpanda cluster.
++
+[NOTE]
+====
+Be sure to set the following cluster-level properties to `false` to disable Tiered Storage for new topics:
+
+* config_ref:cloud_storage_enable_remote_write,false,properties/object-storage-properties[]
+* config_ref:cloud_storage_enable_remote_read,false,properties/object-storage-properties[]
+
+For details, see xref:manage:tiered-storage.adoc#enable-tiered-storage-for-a-cluster[Enable Tiered Storage for a cluster].
+====
+
+- xref:manage:tiered-storage.adoc#configure-object-storage[Configure object storage].
+- You must have an Enterprise license.
+
+To check if you already have a license key applied to your cluster:
+
+[,bash]
+----
+rpk cluster license info
+----
+endif::[]
+
+== Limitations
+
+Not all Redpanda features are available when using Cloud Topics (beta). The following features are not supported in this beta release:
+
+- Kafka compaction
+- Iceberg topics
+- Topic recovery
+- Remote Read Replicas
+
+Also note that Cloud Topics do not work in simple Docker containers unless Cloud storage is set up in them.
+
+IMPORTANT: You should expect to throw away any cluster running Cloud Topics (beta) after it becomes generally available (GA). Redpanda will be unable to continue using data that was generated by the beta version of this feature. Also, any topic specified to be a Cloud Topic cannot subsequently be converted back to a standard Kafka topic.
+
+== Set up Cloud Topics
+
+To set up Cloud Topics:
+
+. Enable the Cloud Topic for your cluster.
+
+. Create the Cloud Topic.
+
+You can make a topic a Cloud Topic only at creation time.
+
+IMPORTANT: Once you create a Cloud Topic, you cannot subsequently convert it back into a standard Kafka topic.
+
+=== Enable Cloud Topics for your cluster
+
+Run the `rpk cluster config set` command, then edit the following required property:
+
+[,properties]
+----
+unstable_beta_feature_cloud_topics_enabled: true
+----
+
+=== Create a Cloud Topic
+
+Run the following command to create a Cloud Topic:
+
+[,bash]
+----
+rpk topic create -c redpanda.cloud_topic.enable=true <new-cloud-topic-name>
+TOPIC                    STATUS
+audit.analytics.may2025  OK
+----
+
+NOTE: For non-replicated cross-AZ traffic (for example, for producers), if your throughput costs remain elevated, you can use xref:../produce-data/leader-pinning.adoc[leader pinning], which positions a topic's partition leader close to the producers, and helps eliminate this traffic.
```

Comment:
"NOTE: In addition to replication, cross-AZ ingress (Producer) and egress (Consumer) traffic can also contribute substantially to cloud networking cots. When running multi-AZ clusters in general, Redpanda strongly recommends using [follower fetching](https://docs.redpanda.com/current/develop/consume-data/follower-fetching/) allowing consumers to avoid crossing network zones, and (where possible) use xref:../produce-data/leader-pinning.adoc[leader pinning], which positions a topic's partition leaders close to producers, providing a similar benefit for ingress traffic.  These features stack additional savings onto the replication cost savings of cloud topics and are useful for all topics in avoiding unnecessary infrastructure costs."